website/static/js/logFeed.js

```diff
@@ -82,6 +82,16 @@ var Log = function(params) {
         }
         return ret;
     });
+
+    //helper function to strip the leading slash for file or folder in log template
+    self.stripLeadingSlash = function(path){
+        return path.replace(/^\//, '');
+    };
+
+    //helper funtion to determine the type for removing in log template
+    self.typeToRemove = function(path){
```

Comment:
This function is not specific to "remove" logs, so the name can be more general. I will rename it to `pathType` when I merge.